src/otx/core/model/module/classification.py

```diff
@@ -215,27 +215,14 @@ def __init__(
             metric=metric,
         )
 
-        if metric:
```

Comment:
For the other colleagues, this changes will be resolved at 
this PR: [https://github.com/openvinotoolkit/training_extensions/pull/2964](https://github.com/openvinotoolkit/training_extensions/pull/2964)